src/scatra/4C_scatra_timint_genalpha.cpp

```diff
@@ -312,12 +312,45 @@ void ScaTra::TimIntGenAlpha::compute_time_derivative()
   // call base class routine
   ScaTraTimIntImpl::compute_time_derivative();
 
+
+  // target map for phidtnp_
+  const auto& tgtMap = phidtnp_->get_map();
+
+  // create temp vector and importer
+  static std::unique_ptr<Core::LinAlg::Vector<double>> phinp_owned, phin_owned;
+  static std::unique_ptr<Core::LinAlg::Import> importer_phinp_, importer_phin_;
```

Comment:
According to our naming convention, variables ending with "_" are class members.

```suggestion
  static std::unique_ptr<Core::LinAlg::Import> importer_phinp, importer_phin;
```